core/mondoo-aws-security.mql.yaml

```diff
@@ -949,7 +913,7 @@ queries:
   - uid: mondoo-aws-security-rds-instance-public-access-check
     title: Checks that all RDS instances are not publicly accessible
     impact: 95
-    mql: "# @msg Found ${length} RDS instance(s) that are publicly accessible:\n# \n# ${{arn name region dbInstanceIdentifier tags}}\naws.rds.dbinstance { publiclyAccessible == false }\n"
```

Comment:
@chris-rock would it make sense for us to add to the linter to detect when a query is quoted out to a string?

Reply:
The linter will not detect that since it is valid yaml. We need to adjust the formatter to format it properly